modules/explorer-api/src/main/scala/org/ergoplatform/explorer/http/api/v0/HttpApiV0.scala

```diff
@@ -17,14 +30,21 @@ object HttpApiV0 {
 
   def apply[
     F[_]: ConcurrentEffect: ContextShift: Timer,
-    D[_]: LiftConnectionIO: ContravariantRaise[*[_], InconsistentDbData]: Monad
+    D[_]: ContravariantRaise[*[_], InconsistentDbData]: ContravariantRaise[*[_], DexErr]: ContravariantRaise[
+      *[_],
+      Base16DecodingFailed
+    ]: ContravariantRaise[
+      *[_],
+      ErgoTreeDeserializationFailed
```

Comment:
Lets try to group these errors into some hierarchy to avoid having so many `ContravariantRaise` in constructor.

Reply:
I made `DexError` a subtype of `RequestProcessingError` and relaxed requirements a lot here and a bit on lower layers. 
Furthermore, we can think of `DecodingError` to be a supertype for `AddressDecodingFailed`, `Base16DecodingFailed`, `ErgoTreeDeserializationFailed`.